### PR TITLE
[IMP] website_jitsi: allow custom server deployment

### PR DESCRIPTION
**Pull request by std-odoo**
Purpose
=======
Allow to customize the domain name of the Jitsi server used
to create chat room on website.

This is done with an `ir.config_parameter`.

Task-2316718

### CODE REVIEW

File: addons/website_jitsi/models/chat_room.py [hunk 1]
```diff
@@ -29,6 +29,9 @@ def _default_name(self):
         "Room Name", required=True, copy=False,
         default=lambda self: self._default_name())
     is_full = fields.Boolean("Full", compute="_compute_is_full")
+    jitsi_server_domain = fields.Char(
+        'Jitsi Server Domain', compute='_compute_jitsi_server_domain',
+        help='The Jitsi server domain can be customized through the settings to use a different server than the default "meet.jit.si"')
```
Comment (sswapnesh) [inline addons/website_jitsi/models/chat_room.py:34]: > can be customized through the settings

Are we going to add Settings (res.config.settings) for this  :thinking: ?

Comment (std-odoo) [reply addons/website_jitsi/models/chat_room.py:34]: Yes, but in master

Comment (std-odoo) [reply addons/website_jitsi/models/chat_room.py:34]: See https://github.com/odoo/odoo/pull/55869

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Pull request closed without merging**